src/makeRobotAccountant.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  const getSum = function(firstAddition) {
```

Comment:
It would be a little bit cleaner to simply return a function without declaring a variable and by using arrow functions